power-apps/multiple-popin/README.md

```diff
@@ -0,0 +1,1864 @@
+# Multiple Popins
+
+This snippet is composed of 4 pop-ins for your applications. Reduce the number of components in your PowerApps apps by always using this snippet.
+
+You can put this snippet in each of your screens.
+
+You can also customize the language.
+
+![multiple-popin](./assets/multiple-popin.png)
+
+
+
+## Authors
+
+Snippet|Author
+--------|---------
+Steve Bourdin | [GitHub](https://github.com/SteveBourdin) ([@steve-bourdin-ab998762](https://www.linkedin.com/in/steve-bourdin-ab998762/) )

```

Comment:
Please change this to:

`Steve Bourdin | [GitHub](https://github.com/SteveBourdin) ([LinkedIn](https://www.linkedin.com/in/steve-bourdin-ab998762/) )`